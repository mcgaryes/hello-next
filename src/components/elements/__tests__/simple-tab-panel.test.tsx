import {render} from '@testing-library/react'
import SimpleTabPanel from "@/elements/simple-tab-panel";

test('Renders simple-tab-panel unchanged', () => {
    const {container} = render(<SimpleTabPanel />)
    expect(container).toMatchSnapshot()
})
