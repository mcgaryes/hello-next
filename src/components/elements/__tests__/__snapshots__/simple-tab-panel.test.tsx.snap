// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders simple-tab-panel unchanged 1`] = `
<div>
  <div />
</div>
`;
